README.md

```diff
@@ -58,12 +58,12 @@ How to Run
 Either install Vega Strike from the binary installer for your platform, if available, or follow the instructions for compiling from source. (`Compiling Vegastrike`, below.)
 
 - First change settings with `bin/vegasettings`
-- Then run `bin/vegastrike` or double-click on the executable file in the bin directory.
+- Then run `bin/vegastrike-engine` or double-click on the executable file in the bin directory.
 
 Vegastrike command line parameters allow for different start settings and are as follows
 
 ```man
-  vegastrike [-ddata_dir] [-px,y,z] [-jsector/starsystem] missionname
+  vegastrike-engine [-ddata_dir] [-px,y,z] [-jsector/starsystem] missionname
```

Comment:
`-ddata_dir` is required for the `vegastrike-engine` executable. It's only optional for the `vegastrike` executable.

Reply:
Good catch

Reply:
Fixed